qiita_pet/handlers/analysis_handlers.py

```diff
@@ -111,6 +111,10 @@ def get(self, analysis_id):
     def post(self, analysis_id):
         analysis_id = int(analysis_id)
         rarefaction_depth = self.get_argument('rarefaction-depth')
+        mdsi = self.get_argument('merge-duplicated-sample-ids', default=False)
+        if mdsi == 'on':
```

Comment:
Not sure about this one. By default it's always False, line above. Remember that if is not "on", the argument doesn't exist.